packages/mobile/src/screens/explore-screen/components/ExploreContent.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useCallback, useMemo } from 'react'
```

Comment:
file could probably use a bit of refactoring itself? here's a quick AI suggestion: 

https://docs.google.com/document/d/1SCdjoNhCS9u5bUwNrENoN-M8g6kt4fDK8zRhbXDeOOQ/edit?usp=sharing

Reply:
Yeah completely agree!